comit/src/btsieve.rs

```diff
@@ -90,7 +90,7 @@ where
             seen_blocks.extend(missed_blocks);
 
             // The duration of this timeout could/should depend on the network
```

Comment:
Shall we tackle the comment above?

Reply:
Definitely, I don't think there is a point in just changing the delay from 1 sec to 30 :D